docs/docusaurus/versioned_docs/version-1.5.0/orc8r/upgrade_1_5.md

```diff
@@ -2,22 +2,129 @@
 id: version-1.5.0-upgrade_1_5
 title: Upgrade to v1.5
 hide_title: true
-original_id: upgrade_1_5
 ---
 
 # Upgrade to v1.5
 
-This guide covers upgrading Orchestrator deployments from v1.4 to v1.5.
+This guide covers upgrading Orchestrator deployments from either v1.3 or v1.4
+to v1.5.
 
 The v1.5 upgrade follows a standard procedure:
 
 - Migrate NMS DB data to orc8r DB
-- etc.
+- Upgrade Terraform modules
+- Bump version numbers
+
+## Prerequisites
+
+Terraform version 0.14.5 is recommended for orc8r upgrades.
+Current terraform scripts are *not* compatible with terraform 0.15.0.
+
+Build and publish the application containers and Helm charts on the head of the
+release branch by following the [Build Orchestrator](https://docs.magmacore.org/docs/orc8r/deploy_build)
+documentation.
+
+If you are using local Terraform state (the default), ensure all Terraform
+state files are located in your working directory before proceeding. This means
+`terraform show` should list existing state, rather than outputting `No state`.
+
+Now fetch the Kubernetes version that your cluster is running. If the AWS CLI
+is configured locally, run
+
+```
+aws eks describe-cluster --name orc8r
+```
+
+and note the `version` listed under `cluster`.
+
+Otherwise, navigate to the AWS UI. Enter service `EKS` and select clusters on
+the left hand side of the screen. Note the `Kubernetes version` for the
+`orc8r` cluster. Use this version to set `cluster_version` below.
 
 ## NMS DB Data Migration
 
-Run on a machine with access to the kubernetes cluster you are using to run Magma.
+As a pre-requisite task to orc8r upgrade to 1.5, the NMS DB migration needs to
+be completed. This migration copies all NMS data to the same database housing
+orc8r data.
+These tables are named
+`Users, Organizations, FeatureFlags, AuditLogEntry, SequelizeMeta`
+A single script is provided to perform this migration for you, and should
+tell you whether or not it was successful. After migration, the tables listed
+above should be present in the orc8r database.
+
+Run on a machine with access to the kubernetes cluster you are using to run
+Magma.
 
 *Paste in shell prompt:*
 
 `wget https://raw.githubusercontent.com/magma/magma/master/nms/app/packages/magmalte/scripts/fuji-upgrade/pre-upgrade-migration.sh && chmod +x pre-upgrade-migration.sh && ./pre-upgrade-migration.sh`
+
+*Terraform main.tf Sanitization*
+
+References to NMS DB need to be removed from the main.tf file.
+Further, the dialect for the orc8r DB needs to be specified.
```

Comment:
minor nit: Change `Further, the dialect for the orc8r DB needs to be specified.` => ```New variable "orc8r_db_dialect" for specifying DB dialect was added in 1.5. This variable's default value is set to postgres. In case of using non postgres db instance, this has to be modified. Common dialects will be `mysql`, `postgres`, and `mariadb`.
See [here](https://sequelize.org/v5/manual/dialects.html) for a list.```